dlang/plugin-impl/src/main/resources/META-INF/plugin.xml

```diff
@@ -29,9 +28,6 @@
     ]]>
     </change-notes>
 
-    <!-- please see http://confluence.jetbrains.com/display/IDEADEV/Build+Number+Ranges for description -->
-    <idea-version since-build="233" until-build="241.*"/>
```

Comment:
as mentioned before, may be better to control this manually. Also, I seem to remember the plugin setting versions to specific builds, so instead of 2023.3 it would result in _233.11799.241_ etc. In turn this can end up not supporting future releases of the same minor.major version.

Reply:
We control it but with the equivalent of the old "patch xml" version. I set the config in the gradle.properties to have to edit only 1 file instead of manually patching the XML